b2 needs helps searching for system-wide kernel/boost-build.jam to avoid
error: "Unable to load b2: could not find 'boost-build.jam'".

Alternatively, set environment variable:
BOOST_BUILD_PATH=${PREFIX}/share/boost-build/kernel b2

Index: tools/build/src/engine/startup.cpp
--- tools/build/src/engine/startup.cpp.orig
+++ tools/build/src/engine/startup.cpp
@@ -201,7 +201,7 @@ bool b2::startup::bootstrap(FRAME *frame)
     {
         const std::string path{
             b2::paths::normalize(
-                b2_exe_path + "/../../share/boost-build/src/kernel/" + boost_build_jam)};
+                "${PREFIX}/share/boost-build/kernel/" + boost_build_jam)};
         if (b2::filesys::is_file(path))
             b2_file_path = path;
     }
